lib/helper.js

```diff
@@ -292,6 +321,15 @@ Helper.prototype.waitForModalTransition = function() {
   }, self.MAX_TIMEOUT, 'should wait for modal to transition');
 };
 
+Helper.prototype.waitForModalToClose = function(modal) {
+  var self = this;
+  browser.wait(function() {
+    return modal.isPresent().then(function(res) {
+      return !res;
+    });
+  }, self.MAX_TIMEOUT, 'should wait for modal to close');
+};
```

Comment:
No, but I don't know where it's used. If we do need it, it needs to be generalized. I'm fine to remove it.